### PR TITLE
ALICLoud RDS parameter checks

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
*removes errant ALI_24 duplicate check.

- Parameter checks on rds for logging, log_connections, log disconnection
- Check to ensure minor version upgrades are enabled

### CODE REVIEW

File: checkov/terraform/checks/resource/alicloud/AbsRDSParameter.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AbsRDSParameter(BaseResourceCheck):
+    def __init__(self, check_id, parameter):
+        name = "Ensure RDS instance has %s enabled" % parameter
```
Comment (gruebel) [inline checkov/terraform/checks/resource/alicloud/AbsRDSParameter.py:7]:
```suggestion
        name = f"Ensure RDS instance has {parameter} enabled"
```
🙂

File: checkov/terraform/checks/resource/alicloud/AbsRDSParameter.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    def scan_resource_conf(self, conf):
+        """
+            Looks at configuration of RDS logging parameters :
+
+        :param conf: alicloud_db_instance configuration
+        :return: <CheckResult>
+        """
+        if conf.get("parameters") and isinstance(conf.get("parameters"), list):
+            params = conf.get("parameters")
+            for param in params:
```
Comment (gruebel) [inline checkov/terraform/checks/resource/alicloud/AbsRDSParameter.py:22]:
```suggestion
        params = conf.get("parameters")
        if params and isinstance(params, list):
            for param in params:
```

File: checkov/terraform/checks/resource/alicloud/RDSInstanceLogsEnabled.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories,CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class RDSInstanceLogsEnabled(BaseResourceCheck):
```
Comment (gruebel) [inline checkov/terraform/checks/resource/alicloud/RDSInstanceLogsEnabled.py:5]: why not use your own `AbsRDSParameter` here? 😄

Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/alicloud/RDSInstanceLogsEnabled.py]: yeah sunk cost fallacy there.

**Review by gruebel — APPROVED** [review]
noice 🥇

**Review by Saarett — APPROVED** [review]

**Pull request merged**